kuksa_apps/node-red/README.md

```diff
@@ -1,14 +1,45 @@
 # Flow for node-red dashboard
 
-After installing [node-red](https://nodered.org/), you can import the json file to receive data from kuksa-val-server. Then you can use the node-red dashboard feature to view data.
+This example uses [node-red](https://nodered.org/) but an installation is not required at your machine, as we are using Docker. You can import the json file to receive data from kuksa-val-server. Then you can use the node-red dashboard feature to view data.
 
 ## Dependencies
+### Node-red extensions
 The following node red extensions are used in the example flows:
 - [node-red-dashboard](https://flows.nodered.org/node/node-red-dashboard)
 - [node-red-contrib-web-worldmap](https://flows.nodered.org/node/node-red-contrib-web-worldmap)
 
+### Installation
+You can use docker to start `node-red` and `kuksa.val` server.
```

Comment:
You can use docker to start `node-red` and `kuksa.val` server.
Make sure docker and docker-compose are available in your system (or install it using "apt install docker.io docker-compose").